Makefile.onload

```diff
@@ -9,7 +9,7 @@ endif
 
 .PHONY: build_dir
 
-ONLOAD_SCRIPTS := $(ONLOAD_TREE)/scripts
+ONLOAD_SCRIPTS := $(shell cd $(ONLOAD_TREE) && /bin/pwd)/scripts
```

Comment:
Very much a nit (and personal preference), but this might be cleaner to represent as `realpath $(ONLOAD_TREE)` instead of the `cd` and `pwd` combo.